### PR TITLE
docs: Suggest gh manual triggers

### PR DESCRIPTION
**Pull request by a7madgamal**

### CODE REVIEW

File: docs/recipes/github-actions.md [hunk 1]
```diff
@@ -67,7 +67,4 @@ To trigger a release, call (with a [Personal Access Tokens](https://help.github.
 $ curl -v -H "Accept: application/vnd.github.everest-preview+json" -H "Authorization: token ${GITHUB_TOKEN}" https://api.github.com/repos/[org-name-or-username]/[repository]/dispatches -d '{ "event_type": "semantic-release" }'
 ```
 
-If you'd like to use a GitHub app to manage this instead of creating a personal access token, you could consider using a project like:
-
-* [Actions Panel](https://www.actionspanel.app/) - A declaratively configured way for triggering GitHub Actions
-* [Action Button](https://github-action-button.web.app/#details) - A simple badge based mechanism for triggering GitHub Actions
+If you'd like to use a GitHub app to manage this instead of creating a personal access token, check github's [Manual Triggers](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/)
```
Comment (gr2m) [inline docs/recipes/github-actions.md:70]: Please do not remove the existing apps. Actions are not yet available for GitHub Enterprise Server users.

Good idea to add the new workflow dispatch trigger though, I would add that paragraph above the existing one, and maybe rephrase the existing one to be recommended for GHES users.

Also maybe rephrase the new paragraph to something like this? 

```suggestion
You can also use [Manual Triggers](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) for GitHub Actions.
```

File: docs/recipes/github-actions.md [hunk 2]
```diff
@@ -50,7 +50,11 @@ To keep `package.json` updated in the `master` branch, [`@semantic-release/git`]
 
 ## Trigger semantic-release on demand
 
-There is a way to trigger semantic-relase on demand. Use [`repository_dispatch`](https://help.github.com/en/articles/events-that-trigger-workflows#external-events-repository_dispatch) event to have control on when to generate a release by making an HTTP request, e.g.:
+### Using GUI:
+You can use [Manual Triggers](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) for GitHub Actions.
+
+### Using HTTP:
+Use [`repository_dispatch`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#repository_dispatch) event to have control on when to generate a release by making an HTTP request, e.g.:
```
Comment (a7madgamal) [inline docs/recipes/github-actions.md:57]: I updated this link also, please confirm

Comment (gr2m) [reply docs/recipes/github-actions.md:57]: Looks good 👍🏼
